tests/src/Functional/ContentProjectRenderTest.php

```diff
@@ -244,9 +247,35 @@ public function testProjectRendering(): void {
           'definition' => '€22,30',
         ],
       ],
-    ];
-    $field_list_html = $description_lists[4]->getHtml();
-    $field_list_assert->assertPattern($fifth_field_list_expected_values, $field_list_html);
+    ], $description_lists[3]->getHtml());
+
+    // Set a project period that is fully in the past.
+    $this->setProjectDateRange($node, '2019-03-07', '2019-03-21');
```

Comment:
setProjectDateRange already calls node->save and drupalGet, making below calls a problem. But actually, the facet that setProjectDateRange contains those is wrong, at least the drupalGet, the node save you can keep in there and remove from here.

As you can already see, this is why its so important to have the method names be descriptive to what they do, because then I don't have to check what it does, and you and me both could have spotted this earlier.